tests/readmes/test_readmes.py

```diff
@@ -97,6 +98,14 @@ def test_script_execution(script, monkeypatch):
     runpy.run_path(str(script), run_name="__main__", init_globals=globs)
 
 
+@pytest.mark.skipif(
+    VirtualMachineError(
+        ValueError(
+            {"code": -32000, "message": "insufficient funds for gas * price + value"}
+        )
+    ),
+    reason="requires more funds for remote accounts",
+)
```

Comment:
Please undo all the changes in this file, ie to "skip if out of funds".

Why: it's more involved than you think; it's got its own issue that I was working on; and I just merged the fix into main.
- The issue: #1109 
- The PR: https://github.com/oceanprotocol/ocean.py/pull/1111
- The merge commit: https://github.com/oceanprotocol/ocean.py/commit/6985a76a6bf194d4fcf540844b075eec0010e660